delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Schemas.scala

```diff
@@ -194,22 +195,25 @@ object Schemas {
     */
   final val moduleType: String = "schema"
 
+  @SuppressWarnings(Array("OptionGet"))
   private[delta] def next(state: SchemaState, event: SchemaEvent): SchemaState = {
 
-    @SuppressWarnings(Array("OptionGet"))
     // It is fine to do it unsafely since we have already computed the graph on evaluation previously in order to validate the schema.
-    def toGraphUnsafe(expanded: ExpandedJsonLd) =
-      expanded.toGraph.toOption.get
+    def toSchemaGraph(expanded: ExpandedJsonLd) =
+      expanded.filterType(nxv.Schema).toGraph.toOption.get
+
+    def toOntologyGraph(expanded: ExpandedJsonLd) =
+      expanded.filterType(nxv.Ontology).toGraph.toOption.get
```

Comment:
as far as I know, we have schemas that also have the type Ontology so this strategy of splitting based on type will not work as it's implemented right now.

Reply:
I didn't find anything like that in neuroshapes. However, I've made the change